index.js

```diff
@@ -1272,6 +1315,8 @@
     ap: ap,
     lift2: lift2,
     lift3: lift3,
+    apFirst: apFirst,
+    apSecond: apSecond,
```

Comment:
Good point, @zhangchiqing. The location you chose is perfect! I'll merge this when I get home. :)